mvcc/kv.go

```diff
@@ -37,7 +49,7 @@ type KV interface {
 	// If `end` is not nil and empty, it gets the keys greater than or equal to key.
 	// Limit limits the number of keys returned.
 	// If the required rev is compacted, ErrCompacted will be returned.
-	Range(key, end []byte, limit, rangeRev int64) (kvs []mvccpb.KeyValue, rev int64, err error)
+	Range(key, end []byte, ro RangeOptions) (r *RangeResult, err error)
```

Comment:
maybe this should be `Range(key []byte, ro RangeOptions)` since it's so similar to the client interface?